{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:19.040Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:19.185Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:19.413Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:20.122Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:20.201Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:34.876Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:35.108Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:35.266Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:35.488Z"}
{"level":"error","message":"Uncaught internal server error. { [MongoError: connect ECONNREFUSED 127.0.0.1:27017]\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:893:11)\n    at exports._exceptionWithHostPort (util.js:916:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1075:14)","timestamp":"2016-05-31T05:53:37.300Z"}
